src/controller/python/chip/clusters/__init__.py

```diff
@@ -32,12 +32,13 @@
                       FixedLabel, FlowMeasurement, FormaldehydeConcentrationMeasurement, GeneralCommissioning, GeneralDiagnostics,
                       GroupKeyManagement, Groups, HepaFilterMonitoring, IcdManagement, Identify, IlluminanceMeasurement,
                       KeypadInput, LaundryWasherControls, LaundryWasherMode, LevelControl, LocalizationConfiguration, LowPower,
-                      MediaInput, MediaPlayback, ModeSelect, NetworkCommissioning, NitrogenDioxideConcentrationMeasurement,
```

Comment:
Could we just don `from .Objects import *`

Reply:
If that works it would be nice, as far as I understand this is a somewhat manual step that needs to be done, hence why it sometimes tend to be missed. I'm wondering if it is because .Objects contains additional stuff which we do not want 🤔

Reply:
https://github.com/project-chip/connectedhomeip/issues/31232.

If we are worries about other stuff leaking from `.Objects` we can restructure it such that all the clusters are in a single file

I haven't started investigating. But just if anyone else takes it on, they can take that issue off my hand :stuck_out_tongue: